### PR TITLE
Hooks for using printer terminal in web & mobile app

### PR DESCRIPTION
**Pull request by aarontuftss**

### CODE REVIEW

File: octoprint_obico/__init__.py [hunk 1]
```diff
@@ -356,6 +358,13 @@ def post_update_to_server(self, data=None):
         self.send_ws_msg_to_server(data)
         self.status_posted_to_server_ts = time.time()
 
+    def post_terminal_feed_msg(self, message):
```
Comment (kennethjiang) [inline octoprint_obico/__init__.py:361]: Have you tried to reuse "passthru" for this purpose? The use case seems to fit the purpose of "passthru" really well. The only reason why "passthru" can't be reused for this purpose is if it'll cause problems on old mobile app versiosn.

File: octoprint_obico/gcode_hooks.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import logging
 import re
 import sys
+import time
```
Comment (kennethjiang) [inline octoprint_obico/gcode_hooks.py:4]: Don't think this is used anywhere, right?

File: octoprint_obico/print_job_tracker.py [hunk 3]
```diff
@@ -67,6 +67,7 @@ def status(self, plugin, status_only=False):
             'status': plugin._printer.get_current_data()
         }
 
+
```
Comment (kennethjiang) [inline octoprint_obico/print_job_tracker.py:70]: Really unncessary.

File: octoprint_obico/__init__.py [hunk 4]
```diff
@@ -86,6 +86,8 @@ def __init__(self):
         self.bailed_because_tsd_plugin_running = False
         self.printer_events_posted = dict()
         self.file_operations = FileOperations(self)
+        self.terminal_feed_sent = None
```
[the remark this answers is not in the document]
Comment (aarontuftss) [reply octoprint_obico/__init__.py]: No, removed in recent commit.

File: octoprint_obico/__init__.py (hunk 1 above)
Comment (aarontuftss) [reply octoprint_obico/__init__.py]: just updated to use passthru - if we use a webview as planned it shouldnt cause an issue, correct?

File: octoprint_obico/gcode_hooks.py (hunk 2 above)
Comment (aarontuftss) [reply octoprint_obico/gcode_hooks.py:4]: it is now

**Pull request merged**